includes/class-wc-stripe-apple-pay-registration.php

```diff
@@ -278,9 +325,7 @@ public function verify_domain_on_new_secret_key( $prev_settings, $settings ) {
 	 * @version 4.5.3
 	 */
 	public function verify_domain_if_needed() {
-		if ( $this->apple_pay_domain_set ) {
-			$this->update_domain_association_file();
-		} else {
+		if ( $this->payment_request ) {
```

Comment:
Why do this in a payment request?

Reply:
This refers to the Payment Request Button gateway setting. We should have been checking for it here to begin with, I believe – my mistake from an earlier PR.

Reply:
Yeah, let's rename it to something like payment_req_btns_enabled or something

Reply:
Rolled this into a new `in_enabled` method and gave it a more clear name in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1357/commits/21cc4b60bf18e1503c731eceda4038c49753251f